images/runtime/php-fpm/template.base.Dockerfile

```diff
@@ -93,7 +93,7 @@ RUN set -eux; \
 #  - https://docs.microsoft.com/en-us/sql/connect/php/installation-tutorial-linux-mac
 #  - https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server
 RUN set -eux; \
-    if [[ $PHP_VERSION == 7.4.* || $PHP_VERSION == 8.0.* ]]; then \
+    if [[ $PHP_VERSION == 7.4.* || $PHP_VERSION == 8.* ]]; then \
```

Comment:
Could you please clarify how updating this will install imagick? Is this related to the OBDC driver mentioned here?

Reply:
It's the pdo_sqlsrv library that 8.1 didn't include, so I simply added it and corresponding tests.

Reply:
OK, and so imagick was already installed separately? I guess I'm just confused as I didn't see where (cf. my other comment).

Reply:
Yes it's a seperate one.